attractions/time-picker/time-picker.svelte

```diff
@@ -121,24 +133,26 @@
       <div class="shown-on-focus">
         <Button noRipple on:click={() => focus = false}>close the time picker</Button>
       </div>
-      <Label>Hours</Label>
+      <Label>{hoursLabel}</Label>
       <div class="column">
-        {#each hourValues as value}
-        <Button on:click={() => setHours(value + 12 * (f12hours && currentAmPm === 'PM' ^ value === 12))}>
-          {value.toString().padStart(2, '0')}
-        </Button>
+        {#each hourValues as hour}
+          <Button
+            on:click={() => setHours(hour + 12 * (f12hours && (currentAmPm === 'PM') ^ (value === 12)))}
+            filled={matchesCurrentHour(hour, value)}>
+            {hour.toString().padStart(2, '0')}
+          </Button>
         {/each}
       </div>
-      <Label>Minutes</Label>
+       <Label>{minutesLabel}</Label>
       <div class="column">
-        {#each minuteValues as value}
-        <Button on:click={() => setMinutes(value)}>
-          {value.toString().padStart(2, '0')}
-        </Button>
+        {#each minuteValues as mins}
+          <Button on:click={() => setMinutes(mins)} filled={value && mins === value.getMinutes()}>
```

Comment:
Not really digging this way of emphasis. Filled buttons have a semantic value, I'm afraid they cannot simply be used for emphasis like this.
We've been thinking about this emphasis thing too, haven't come up with anything reasonable yet. Maybe use the Dot component to mark the current time?

Reply:
I get your point. I can try something else. I suppose most date/time pickers have a 'selected' style. But I can see what the dot would look like.

Reply:
I just noticed there is a selected prop on the Button. Would this be appropriate?  The Dot looks a bit odd to my eye.

Reply:
Well, it kinda clashes with the focus styles, so that might become confusing. And now that I had a second though about it, is it really worth highlighting the current time? Given that there's a "now" button that can automatically select the current time.